org.adempiere.base/src/org/compiere/model/PO.java

```diff
@@ -1419,6 +1484,16 @@ public boolean load (String trxName, String ... virtualColumns) {
 	 */
 	protected boolean load (String uuID, String trxName, String ... virtualColumns)
```

Comment:
this is a mistake from earlier revision, needs to rename it to something else as it is getting dangerously close to `public boolean load (String trxName, String ... virtualColumns)`

Reply:
Perhaps rename it to loadByUUOrID. Needs to fix the javadoc too to reflect that if uuID is null, it will reload using current IDs values.

Reply:
We need t fix `public boolean load (String trxName, String ... virtualColumns)` too. This is commonly use to reload PO during or after trx and with the new design change, it must works with table that only has _UU key too.

Reply:
For convenience, probably should add a new method to mtable, for e.g IsUUIDTable() to easily identify table that don't have ID key and is using UU as key.